packages/mui-system/src/Container/ContainerProps.ts

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { OverrideProps } from '@mui/types';
+import { SxProps } from '../styleFunctionSx';
+import { Theme, Breakpoint } from '../createTheme';
+import { ContainerClasses } from './containerClasses';
+
+export interface ContainerTypeMap<P = {}, D extends React.ElementType = 'div'> {
+  props: P & {
+    children?: React.ReactNode;
+    /**
+     * Override or extend the styles applied to the component.
+     */
+    classes?: Partial<ContainerClasses>;
```

Comment:
```suggestion
```
I think system should not accept `classes` prop also.

Reply:
Let me check how this will align with the Material UI implementation

Reply:
If we want to not support classes in the system it would mean that Material UI's Container will need to create additional wrapper. I would leave as is for now, remove the classes prop from the Joy props' interface, and we can revisit this in v6 again. What do you think?

Reply:
That's fine for me!